src/sentry/eventstore/models.py

```diff
@@ -753,6 +756,8 @@ def __getitem__(self, name: str) -> str:
             return cast(str, self.event.organization.slug)
         elif name == "title":
             return self.event.title
+        elif name == "issueType":
+            return cast(str, GROUP_TYPE_TO_TEXT[self.event.group.issue_type])
```

Comment:
Let's use `.get` here and default to some value. Just for in the future if we add some new issue type.

Alternatively, maybe write a unit test that validates that all `GroupType` values have an entry in that dict?